lib/Rex/Helper/Run.pm

```diff
@@ -157,7 +157,7 @@ sub i_exec_nohup {
   my $exec   = Rex::Interface::Exec->create;
   my $quoter = Net::OpenSSH::ShellQuoter->quoter( $exec->shell->name );
 
-  my $_cmd_str = "$cmd " . join( " ", map { $_ = $quoter->quote($_) } @args );
+  my $_cmd_str = "$cmd " . join( " ", map { $_ = $quoter->quote($_) } @args ); ## no critic ProhibitMutatingListFunctions
```

Comment:
`@args` isn't used in the remainder of the function, which leads to a similar argument as above:

The code as it is there now, is equivalent to:

```perl
@args = map { "$suffix:$_" } @args;
```

due to the fact that the mutated value of `@args` isn't used. I think it's better, even for the scope of this PR not to suppress but to fix this instance.